apis/Google.Cloud.Redis.V1/Google.Cloud.Redis.V1/CloudRedisGrpc.cs

```diff
@@ -536,142 +486,6 @@ protected CloudRedisClient(ClientBaseConfiguration configuration) : base(configu
         return CallInvoker.AsyncUnaryCall(__Method_UpdateInstance, null, options, request);
       }
       /// <summary>
-      /// Import a Redis RDB snapshot file from GCS into a Redis instance.
-      ///
-      /// Redis may stop serving during this operation. Instance state will be
-      /// IMPORTING for entire operation. When complete, the instance will contain
-      /// only data from the imported file.
-      ///
-      /// The returned operation is automatically deleted after a few hours, so
-      /// there is no need to call DeleteOperation.
-      /// </summary>
-      /// <param name="request">The request to send to the server.</param>
-      /// <param name="headers">The initial metadata to send with the call. This parameter is optional.</param>
-      /// <param name="deadline">An optional deadline for the call. The call will be cancelled if deadline is hit.</param>
-      /// <param name="cancellationToken">An optional token for canceling the call.</param>
-      /// <returns>The response received from the server.</returns>
-      public virtual global::Google.LongRunning.Operation ImportInstance(global::Google.Cloud.Redis.V1.ImportInstanceRequest request, grpc::Metadata headers = null, global::System.DateTime? deadline = null, global::System.Threading.CancellationToken cancellationToken = default(global::System.Threading.CancellationToken))
```

Comment:
Why are all these changes in a GA version of the API? (I know we've only released a beta, but this looks like a GAed api version)